tools/validators/instance_validator/validate/subscriber.py

```diff
@@ -31,10 +31,9 @@ class Subscriber(object):
     service_account_info: service account information from the GCP project.
```

Comment:
can you please add `(optional) when not provided, defaults to gcloud credentials` or similar